### PR TITLE
Add toggle for campaign section in cart

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add animation to "Switch" component

Make Switch component global

Add toggle for campaign section in cart

## Justify why they are needed

Design updates for the cart from Wille

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by alebedev — APPROVED** [review]
Nice!

File: apps/store/src/components/CartInventory/CampaignsSection.tsx [hunk 1]
```diff
@@ -74,6 +78,26 @@ export const CampaignsSection = ({ cartId, campaigns }: Props) => {
...
+    <Space y={1}>
+      <SpaceBetween>
+        <Heading as="h3" variant="standard.18">
+          {t('CAMPAIGN_CODE_HEADING')}
+        </Heading>
+
+        <Switch
+          checked={open}
+          onCheckedChange={handleCheckedChange}
+          disabled={campaigns.length > 0}
```
Comment (gustaveen) [inline apps/store/src/components/CartInventory/CampaignsSection.tsx:94]: A bit unsure about the UX with the toggle. Makes sense to have it disabled when you add a discount but I wonder if it's understandable how to remove it. Let's evaluate 👍

**Review by gustaveen — APPROVED** [review]
Added via Giphy

File: apps/store/src/components/CartInventory/CampaignsSection.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/CartInventory/CampaignsSection.tsx:94]: I will show William!

**Review by guilhermespopolin — APPROVED** [review]
Nice :)

**Pull request merged**